### PR TITLE
Display rebuild link in local case only

### PR DESCRIPTION
**Pull request by YeeSkywalker**
Close: #289 
Sign off: Yee Lin

### CODE REVIEW

**Comment by YeeSkywalker** [comment]
By following the instruction of Shelley's reply [Display rebuild link in local case only #286 #289](https://github.com/adoptium/TKG/pull/289#issuecomment-1063498823), the local test result is shown as below:

```console
jdk_jdi_jdk8_0_FAILED

Nothing to be done for teardown.
jdk_jdi_jdk8_0 Finish Time: Wed Apr  6 13:13:30 2022 Epoch Time (ms): 1649272410833
cd "/Users/yeeskywalker/aqa-tests/TKG/../TKG/scripts"; \
	perl "resultsSum.pl" --failuremk="/Users/yeeskywalker/aqa-tests/TKG/../TKG/failedtargets.mk" --resultFile="/Users/yeeskywalker/aqa-tests/TKG/../TKG/output_16492723099836/TestTargetResult" --platFile="/Users/yeeskywalker/aqa-tests/TKG/../TKG/resources/buildPlatformMap.properties" --diagnostic=failure --jdkVersion=8 --jdkImpl=hotspot --jdkVendor="Temurin" --spec=osx_x86-64 --buildList=openjdk --customTarget= --testTarget=jdk_jdi_jdk8_0 --tapPath=/Users/yeeskywalker/aqa-tests/TKG/../TKG/output_16492723099836/ --tapName=


TEST TARGETS SUMMARY
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
FAILED test targets:
	jdk_jdi_jdk8_0

TOTAL: 1   EXECUTED: 1   PASSED: 0   FAILED: 1   DISABLED: 0   SKIPPED: 0
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
To rebuild the failed test in a jenkins job, copy the following link and fill out the <Jenkins URL> and <FAILED test target>:
<Jenkins URL>/parambuild/?JDK_VERSION=8&JDK_IMPL=hotspot&JDK_VENDOR=Temurin&BUILD_LIST=openjdk&PLATFORM=x86-64_mac&TARGET=<FAILED test target>

For example, to rebuild the failed tests in <Jenkins URL>=https://ci.adoptopenjdk.net/job/Grinder, use the following links:
https://ci.adoptopenjdk.net/job/Grinder/parambuild/?JDK_VERSION=8&JDK_IMPL=hotspot&JDK_VENDOR=Temurin&BUILD_LIST=openjdk&PLATFORM=x86-64_mac&TARGET=jdk_jdi_jdk8_0
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

_jdk_jdi_jdk8_0 done

```

**Comment by YeeSkywalker** [comment]
@llxia Refer to [PR/289](https://github.com/adoptium/TKG/pull/289). Please review when you are free, thank you for your time in advance!

File: scripts/resultsSum.pl [hunk 1]
```diff
@@ -260,47 +260,26 @@ sub resultReporter {
...
-		my $rebuildLinkBase = "parambuild/?JDK_VERSION=$jdkVersion&JDK_IMPL=$jdkImpl$vendorParam$buildParam$platformParam$customTargetParam$jenkinsParam";
-		print "To rebuild the failed test in a jenkins job, copy the following link and fill out the <Jenkins URL> and <FAILED test target>:\n";
-		print "<Jenkins URL>/$rebuildLinkBase&TARGET=<FAILED test target>\n\n";
-		print "For example, to rebuild the failed tests in <Jenkins URL>=${hudsonUrl}job/Grinder, use the following links:\n";
+			my $jenkinsParam = "";
+			my $vendorParam =  "";
+			if ($jdkVendor ne '') {
+				$vendorParam = "&JDK_VENDOR=" . $jdkVendor;
+			}
+			my $rebuildLinkBase = "parambuild/?JDK_VERSION=$jdkVersion&JDK_IMPL=$jdkImpl$vendorParam$buildParam$platformParam$customTargetParam$jenkinsParam";
```
Comment (llxia) [inline scripts/resultsSum.pl:268]: Please remove jenkinsParam from line263 and line268

**Review by llxia — APPROVED** [review]
Thanks @YeeSkywalker

**Review by smlambert — APPROVED** [review]
LGTM

**Pull request merged**